Update ZooFooter links to reflect new FEM pages

goplayoutside3 opened the pull request:
❌ Do not merge until launch of new About and Get Involved pages.

This corresponds to https://github.com/zooniverse/Panoptes-Front-End/pull/7157.

## Package
lib-react-components

## Linked Issue and/or Talk Post
Closes: https://github.com/zooniverse/front-end-monorepo/issues/6025

## Describe your changes
- Update ZooFooter links to match [Figma](https://www.figma.com/design/6rXVi6arrGTmoM08x9tpex/UI-Kit?node-id=5-409&m=dev&t=MGdDZm4sLEz9aSXP-1) and link to new FEM pages
- I had to edit several translations dictionaries, so I'll follow-up this PR with related changes in Lokalise

- Also snuck in an update from twitter logo to X logo in both the ZooFooter and ConnectWithProject.

## How to Review
- Run lib-react-components Storybook to see the ZooFooter
- Run app-root or app-project locally and visit any page

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [x] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [x] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## Post-merge
- [ ] This PR adds translations keys to English dictionary(s). See guidance for pulling new keys to Lokalise [here](https://github.com/zooniverse/how-to-zooniverse/blob/master/Translations/lokalise.md#lokalise-and-fem).

goplayoutside3 commented:
@seanmiller26 Here's a screenshot of the PR changes. Just wanted to double-check the intentional decision to remove links to the blogs?

<img width="1349" alt="Screenshot 2024-08-22 at 11 51 30 AM" src="https://github.com/user-attachments/assets/4a4842ca-c610-4524-807b-2d39c7817e1f">

seanmiller26 commented:
Oversight on my part. There has been a lot of discussion on which sites to include or exclude. Let me confirm with Alisa and I'll report back here.

seanmiller26 commented:
Confirmed that we are actively using the currently shown social media above ^. Let's keep those three the same but use the new 'X' logo for X FKA Twitter.

goplayoutside3 commented:
I changed Twitter --> X , but my first question was about the the News section. It exists on production, but not in the Figma file so I deleted it in this PR, unless we want to keep the links to Zooniverse blogs.

Production:
<img width="148" alt="Screenshot 2024-08-23 at 9 50 41 AM" src="https://github.com/user-attachments/assets/20acb955-f008-4d7e-9fbb-1de605c39e2b">

seanmiller26 commented:
Oops, my mistake! Yes ok to remove from the footer.

goplayoutside3 commented:
@mcbouslog tagging you as a reviewer even though this is considered a draft until ready to launch the About and Get Involved pages. Please review when time, and we'll coordinate merging the week of Sept 10.

goplayoutside3 commented on packages/app-project/src/shared/components/ConnectWithProject/ConnectWithProject.mock.js:
```diff
@@ -58,13 +58,6 @@ export const ConnectWithProjectMock = {
         site: 'tumblr.com',
         url: 'https://tumblr.com/test-project',
       },
-      {
-        name: 'Twitter',
-        label: '',
-        path: 'test-project',
-        site: 'twitter.com',
-        url: 'https://twitter.com/test-project',
```
Deleting this mock because I've left the translation strings as 'twitter' and just changed the icon in ConnectWithProject.

mcbouslog reviewed (approved):
Changes LGTM! Confirmed locally lib-react-components storybook and app-root Footer looks as expected 👍 .

Noting Figma includes "Best practices" (lowercase p), but I think these changes are what is intended ✅ .
